packages/cashier/src/containers/routes/route-with-sub-routes.tsx

```diff
@@ -22,26 +21,19 @@ type TDefaultSubroute = TRoute | undefined;
 const RouteWithSubRoutes = (route: TRouteWithSubRoutesProps) => {
     const renderFactory = (props: RouteComponentProps) => {
         let result = null;
-        if (route.component === Redirect) {
-            let to = route.to;
-
-            // This if clause has been added just to remove '/index' from url in localhost env.
-            if (route.path === routes.index) {
-                const { location } = props;
-                to = location.pathname.toLowerCase().replace(route.path, '');
-            }
-            result = <Redirect to={to} />;
-        } else if (route.is_authenticated && !route.is_logging_in && !route.is_logged_in) {
```

Comment:
remove this part. seems like `route.component` can't be `Redirect` component. pls correct me if i'm wrong